R/check_args.R

```diff
@@ -131,7 +146,32 @@ Neither sd nor shape found: ", print(x)))
 one of a standard deviation (sd) or a shape parameter (shape).
 Both sd and shape found: ", print(x)))
     }
+  }
+
+  if(x$dist == "norm"){
 
+    assertthat::assert_that(assertthat::has_name(x, 'mean'))
+    assertthat::assert_that(assertthat::is.number(x$mean))
+    assertthat::assert_that(assertthat::is.number(x$sd))
+    if(x$sd <= 0) {
+      stop(paste0('Standard deviation for normal distribution must be positive',
+                  ' (currently sd = ',x$sd,'). ABORTING.'))
+    }
+  }
+
+  if(x$dist == "lnorm"){
+
```

Comment:
i think we need assertions here to check that these dist lists have required names (`meanlog` and `sdlog` or `mean` and `sd`).